summer_staj.py

```diff
@@ -44,11 +44,11 @@
 with some other information.
 """
 
-class first_spider(scrapy.Spider):
-    name = 'first_spider'
+class FirstSpider(scrapy.Spider):

```

Comment:
the convention is that file names and variables use small caps letters and underscores, and class names use no underscores and capitalised first letter for words in class names